crashtracker/src/stacktrace.rs

```diff
@@ -58,6 +97,19 @@ mod unix {
     }
 
     impl StackFrame {
+        pub fn normalize_ip(&mut self, normalizer: &Normalizer, pid: Pid) -> anyhow::Result<()> {
+            if let Some(ip) = &self.ip {
+                let ip = ip.trim_start_matches("0x");
+                let ip = u64::from_str_radix(ip, 16)?;
+                let normed = normalizer.normalize_user_addrs(pid, &[ip])?;
```

Comment:
Makes sense to me based on my reading of the blazesym docs - https://docs.rs/blazesym/latest/i686-unknown-linux-gnu/blazesym/normalize/struct.Normalizer.html